src/tqec/templates/display.py

```diff
@@ -29,6 +30,74 @@ def display_template(
         print()
 
 
+def display_templates_ascii(
+    templates: ComposedTemplate,
```

Comment:
I would name that parameter `template` (singular), but this is a very small comment.

Reply:
I took the name from `display_templates_svg()`.
I can change both, or just the name in `display_templates_ascii()`.

Reply:
This is very minor, let's leave it like that for the moment as it does not impact external API.